Documentation/ceph-disaster-recovery.md

```diff
@@ -98,75 +98,77 @@ Patch the `rook-ceph-mon-b` Deployment to stop this mon working without deleting
 
 ```console
 kubectl -n rook-ceph patch deployment rook-ceph-mon-b -p '[{"op":"remove", "path":"/spec/template/spec/containers/0/livenessProbe"}]'
+
 kubectl -n rook-ceph patch deployment rook-ceph-mon-b -p '{"spec": {"template": {"spec": {"containers": [{"name": "mon", "command": ["sleep", "infinity"], "args": []}]}}}}'
 ```
 
 Connect to the pod of a healthy mon and run the following commands.
 
 ```console
 kubectl -n rook-ceph exec -it <mon-pod> bash
-
-# set a few simple variables
-cluster_namespace=rook-ceph
-good_mon_id=b
-monmap_path=/tmp/monmap
-
-# extract the monmap to a file, by pasting the ceph mon command
-# from the good mon deployment and adding the
-# `--extract-monmap=${monmap_path}` flag
-ceph-mon \
-    --fsid=41a537f2-f282-428e-989f-a9e07be32e47 \
-    --keyring=/etc/ceph/keyring-store/keyring \
-    --log-to-stderr=true \
-    --err-to-stderr=true \
-    --mon-cluster-log-to-stderr=true \
-    --log-stderr-prefix=debug \
-    --default-log-to-file=false \
-    --default-mon-cluster-log-to-file=false \
-    --mon-host=$ROOK_CEPH_MON_HOST \
-    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
-    --id=b \
-    --setuser=ceph \
-    --setgroup=ceph \
-    --foreground \
-    --public-addr=10.100.13.242 \
-    --setuser-match-path=/var/lib/ceph/mon/ceph-b/store.db \
-    --public-bind-addr=$ROOK_POD_IP \
-    --extract-monmap=${monmap_path}
-
-# review the contents of the monmap
-monmaptool --print /tmp/monmap
-
-# remove the bad mon(s) from the monmap
-monmaptool ${monmap_path} --rm <bad_mon>
-
-# in this example we remove mon0 and mon2:
-monmaptool ${monmap_path} --rm a
-monmaptool ${monmap_path} --rm c
-
-# inject the modified monmap into the good mon, by pasting
-# the ceph mon command and adding the
-# `--inject-monmap=${monmap_path}` flag, like this
-ceph-mon \
-    --fsid=41a537f2-f282-428e-989f-a9e07be32e47 \
-    --keyring=/etc/ceph/keyring-store/keyring \
-    --log-to-stderr=true \
-    --err-to-stderr=true \
-    --mon-cluster-log-to-stderr=true \
-    --log-stderr-prefix=debug \
-    --default-log-to-file=false \
-    --default-mon-cluster-log-to-file=false \
-    --mon-host=$ROOK_CEPH_MON_HOST \
-    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
-    --id=b \
-    --setuser=ceph \
-    --setgroup=ceph \
-    --foreground \
-    --public-addr=10.100.13.242 \
-    --setuser-match-path=/var/lib/ceph/mon/ceph-b/store.db \
-    --public-bind-addr=$ROOK_POD_IP \
-    --inject-monmap=${monmap_path}
 ```
+>```
```

Comment:
Let's revert this section. These are commands to run in the toolbox, rather than output that requires the `>`